websites/N/Navidrome/metadata.json

```
@@ -0,0 +1,23 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.9",
+	"author": {
+		"id": "263735224221827072",
+		"name": "alpershal"
+	},
+	"service": "Navidrome",
+	"description": {
+		"en": "Navidrome is a self-hosted, open source music server and streamer. It gives you freedom to listen to your music collection from any browser or mobile device."
+	},
+	"url": "navidrome.org",
```

Website supports www subdomain
```suggestion
	"url": [
		"navidrome.org",
		"www.navidrome.org"
	],
```